src/test/kotlin/org/opensearch/replication/integ/rest/StartReplicationIT.kt

```diff
@@ -1221,6 +1221,12 @@ class StartReplicationIT: MultiClusterRestTestCase() {
     }
 
     fun `test that wait_for_active_shards setting is updated on follower through start replication api`() {
+
+        //Ignore this test if clusters dont have multiple nodes
+        if(!isMultiNodeClusterConfiguration){
```

Comment:
we will need to fix this, Can we create a backlog and attach to the method rather than skipping?

Reply:
ack

Reply:
https://github.com/opensearch-project/cross-cluster-replication/issues/1079